StaticAnalyzer/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 2.0.1 on 2018-02-08 19:50
```

Comment:
Let's not have the migration file. Remove it altogether. This is known to break existing users setup.

Reply:
deleted